Kickstarter-iOS/Views/Controllers/ManagePledgeViewController.swift

```diff
@@ -245,8 +238,8 @@ final class ManagePledgeViewController: UIViewController, MessageBannerViewContr
 
   // MARK: - Configuration
 
-  func configureWith(project: Project, reward: Reward) {
-    self.viewModel.inputs.configureWith(project, reward: reward)
+  func configureWith(project: Project) {
```

Comment:
We can actually grab the `reward` directly from the `backing` and the project.

Reply:
This centralizes how we retrieve the reward when the project refreshes (the reward is now always taken from the `backing`)